pkg/webhook/board_update.go

```
@@ -18,27 +19,44 @@ func (h *boardUpdate) EventTypesHandled() []string {
 }
 
 type column struct {
-	name string
-	id   string
+	name        string
+	id          string
+	postProcess bool
 }
 
 var stateMapping = map[string]column{}
 
+var postProcessing = make(map[string]column)
+
 var zenHubApi = "https://api.zenhub.io"
 
 var regex = regexp.MustCompile("(?mi)(?:clos(?:e[sd]?|ing)|fix(?:e[sd]|ing))[^\\s]*\\s+#(?P<issue>[0-9]+)")
 
+var doneColumn = &column{}
+
 func (h *boardUpdate) HandleEvent(eventObject interface{}, gh *github.Client, config config.RepoConfig, logger *zap.Logger) error {
 
 	// initialise from config if needed
 	if len(stateMapping) == 0 {
+
+		logger.Info("Initialising state mappings ...")
+
 		for _, col := range config.Board.Columns {
-			c := column{col.Name, col.Id}
+			c := column{col.Name, col.Id, col.PostProcess}
+
+			if c.postProcess { // the first one flagged as post process will act as doneColumn
```

for me, the comment doesn't fit the behaviour as it looks like that the last one wins.

---

some nit-pick: Is 'postProcess' as a flag not more about an implementation detail, so not such much suited as a key in a configuration file ? Just in progress of reviewing but would be maybe better *what* postprocessing is happening if this flag is set. Maybe something like "isDoneColumn" ?

Also, maybe throw an error if more than one is flagged with this flag.

---

Actually I would it probably name it a bit even more neutral, like `boolean postMergePipeline` to indicate that this pipeline (i think this is correct naming for a column in ZenHub, isn't it ?) is the one used to move the issue to when a PR is merged.

---

+1